arbiter/arbiter-deeplearning4j/src/main/java/org/deeplearning4j/arbiter/layers/Deconvolution2DLayerSpace.java

```diff
@@ -21,15 +21,19 @@
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
+import org.deeplearning4j.arbiter.optimize.api.ParameterSpace;
+import org.deeplearning4j.arbiter.optimize.parameter.FixedValue;
 import org.deeplearning4j.nn.conf.layers.Deconvolution2D;
 
 @Data
 @EqualsAndHashCode(callSuper = true)
 @NoArgsConstructor(access = AccessLevel.PROTECTED) //For Jackson JSON/YAML deserialization
 public class Deconvolution2DLayerSpace extends BaseConvolutionLayerSpace<Deconvolution2D> {
+    protected ParameterSpace<int[]> dilation;
```

Comment:
Move this (and all dilation-related functionality - setLayerOptionsBuilder etc) to BaseConvolutionLayerSpace, it applies to all conv layers.

Though looks like we still need it again in SubsamplingLayerSpace... that doesn't extend BaseConvolutionLayerSpace for some reason. Let's not refactor it now though...

Reply:
ConvolutionLayerSpace also doesn't extend BaseConvolutionLayerSpace. Fix in next release.